ra2ce/analyses/indirect/analyses_indirect.py

```diff
@@ -338,18 +340,28 @@ def multi_link_redundancy(self, graph: dict, analysis: AnalysisSectionIndirect):
                 u, v, k, edata = edges
 
                 if nx.has_path(graph, u, v):
-                    alt_dist = nx.dijkstra_path_length(graph, u, v, weight=analysis.weighing)
+                    alt_dist = nx.dijkstra_path_length(
+                        graph, u, v, weight=analysis.weighing
+                    )
```

Comment:
Supposedly, it does. Nothing changed here.

Reply:
Supposedly, it does. Nothing changed here.

Reply:
well I'm asking because we are delegating this to the `networkx` library, and I doubt they accept `graph: GraphFilesCollection` as input, so I believe the signature of this method ( `multilink_redundancy`). I would like to go through this code together though.

Reply:
I'm not sure which to use: nx.Graph or nx.MultiGraph. Can you tell the difference?

Reply:
As far as I recall we either use a `MultiDiGraph` or a `MultiGraph`, in either way this is already more clarifying, thanks :)